redirects-list.json

```diff
@@ -29,7 +29,9 @@
 
   "^/doc/use-cases/data-and-model-files-versioning/?$                                     /doc/use-cases/versioning-data-and-model-files",
   "^/doc/user-guide/dvc-file-format$                                                      /doc/user-guide/dvc-files-and-directories",
-  "^/doc/user-guide/updating-tracked-files$                                               /doc/user-guide/how-to/update-tracked-files",
+  "^/doc/user-guide/updating-tracked-files$                                               /doc/user-guide/how-to/update-tracked-data",
+  "^/doc/user-guide/how-to/update-tracked-files$                                          /doc/user-guide/how-to/update-tracked-data",
+  "^/doc/user-guide/merge-conflicts$                                                      /doc/user-guide/how-to/merge-conflicts",
```

Comment:
Note for self:

- [ ] Remember to request re-indexing of these 2 new URLs in search.google.com/u/1/search-console

Reply:
Oops, can't right now: https://support.google.com/webmasters/answer/6211453#url_inspection

Will try in a few days...